deploy/split-infra-tenant/020-autorization.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubevirt-csi-node-sa
+  namespace: kubevirt-csi-driver
+---
+apiVersion: v1
```

Comment:
Maybe split these into two files? one for the infra and one for the tenant. The infra doesn't need the node rbac and the tenant doesn't need the controller rbac.

Reply:
yep, makes sense

Reply:
fixed